java/src/jmri/jmrit/cabsignals/CabSignalTableModel.java

```diff
@@ -353,7 +352,7 @@ private void chngblockdir(int row){
         log.debug(" Direction to reverse :{}", Path.decodeDirection(olddirection) );
         
         if (olddirection==0){
-            log.debug("No direction found, setting North East.");
+            log.debug("No direction found, setting to North, East");
```

Comment:
Sorry to be a pain but could we change this to NorthEast to confirm that it's applying a single direction?
Imho the comma looks like 2 directions.
Thanks, Steve.